spec/event_spec.rb

```diff
@@ -96,6 +96,18 @@
     end
   end
 
+  describe '.parse' do
```

Comment:
this is already covered through the parser_spec. could remove it for DRYness reasons...